flink-connectors/flink-connector-pulsar/src/test/java/org/apache/flink/connector/pulsar/source/PulsarSourceITCase.java

```diff
@@ -33,8 +33,10 @@
 import org.apache.flink.testutils.junit.FailsOnJava11;
 
 import org.junit.experimental.categories.Category;
+import org.junit.jupiter.api.Disabled;
 
 /** Unite test class for {@link PulsarSource}. */
+@Disabled("Could not reproduce on local machine.")
```

Comment:
Will you create a follow-up Jira ticket to address this issue? We should not disable a test without any reference on why and what.

Reply:
+1 This is the heaviest and most unstable test of Pulsar connector.

If you simply disable it, you'd better do it in a separate patch with a reference. Otherwise, I'm unconfident to other changes in this patch.

Reply:
Confident or unconfident? I think you approved this pull request earlier?

> you'd better do it in a separate patch with a reference
I agree. It would be nice to have a JIRA ticket to re-enable this, mention the JIRA in the `@Disabled` annotation and do this in a separate commit (but it can be in this PR)

Reply:
> unconfident

There're several follow-up commits including tests disabled. I don't think we should mix up a test fix with test disable.

It seems the current status stuck due to failure cannot reproduce locally. I'll try to help on this point this week.

Reply:
Thanks for clarifying. I didn't check the commits after your approval in detail.